exporter/elasticsearchexporter/metadata.yaml

```diff
@@ -87,6 +87,17 @@ telemetry:
       sum:
         value_type: int
         monotonic: true
+    elasticsearch.bulk_requests.latency:
+      prefix: otelcol.
+      stability:
+        level: alpha
+      enabled: true
+      description: Latency of Elasticsearch bulk operations in seconds.
+      unit: s
+      histogram:
+        value_type: double
+        bucket_boundaries: [0, 0.005, 0.010, 0.025, 0.050, 0.075, 0.100, 0.250, 0.500, 0.750, 1, 2.5, 5, 7.5, 10, 25, 50, 75, 100, 250, 500, 750, 1000, 2500, 5000, 7500, 10000]
+      attributes: [outcome, http_status_code]
```

Comment:
non-blocking: i still think `http_status_code` is wrong, but let's fix it for all metrics once and for all in another PR

Reply:
Ahh, right. I forgot to update the docs 🤦 . Let's do it in this PR itself.

Reply:
Done.